app/schemas/USER.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
FYI, when you use google auth, you get access to a ton of data straight from the API. Not sure if you've looked into this, but it might be helpful is designing the data model.